llumnix/entrypoints/llumnix_utils.py

```diff
@@ -158,23 +159,28 @@ def init_request_output_queue() -> RayQueue:
     return request_output_queue
 
 def init_llumnix_components(engine_manager_args: EngineManagerArgs,
-                            engine_args: AsyncEngineArgs) -> Tuple[LLMEngineManager, List[Llumlet], RayQueue]:
+                            engine_args,
+                            node_id: str) -> Tuple[LLMEngineManager, List[Llumlet], RayQueue]:
     assert engine_args.engine_use_ray and engine_args.worker_use_ray, \
             ("In Llumnix, engine and worker must be ray actor in orther to run step and migrate concurrently.")
-
     engine_manager = init_manager(engine_manager_args)
-    logger.info("Init LLMEngineManager done")
-    instance_ids, llumlets = init_llumlets(engine_manager_args, engine_args)
+    if not engine_manager_args.init_instance_in_manager:
```

Comment:
maybe we can add a parameters checker when parse arguments in api_server.
don't do this in lazy model

Reply:
Good suggestion, we can create a seperate pr to support parameters checker in llumnix.